scripts/npm/android-arm64/package.json

```diff
@@ -12,6 +12,9 @@
         "swc.android-arm64.node",
         "swc"
     ],
+    "bin": {
```

Comment:
this actually brings me a question: where do we use binary for `android`? and do we have to support /cli for this target?

Reply:
Hmm... I'm not sure. Maybe termux?

cc @Brooooooklyn Do you have any idea?

Reply:
Yes, seems like there are many termux users

Reply:
That.. is surprising. I'm curious if we can reduce some of matrix including this target. If we say many, do we know how many are there?

Reply:
(either cases, this PR won't deprecate anything yet)